src/Xamarin.Android.Build.Tasks/Tasks/Aapt2.cs

```diff
@@ -34,7 +34,7 @@ public class Aapt2 : AsyncTask {
 
 		protected string ResourceDirectoryFullPath (string resourceDirectory)
 		{
-			return (Path.IsPathRooted (resourceDirectory) ? resourceDirectory : Path.Combine (WorkingDirectory, resourceDirectory)).TrimEnd ('\\');
+			return (Path.IsPathRooted (resourceDirectory) ? resourceDirectory : resourceDirectory).TrimEnd ('\\');
```

Comment:
Doesn't this line thus become `resourceDirectory.TrimEnd('\\')`?

Reply:
Now that I think about it. we can't do this.
The problem is the MSBuild may well change the WorkingDirectory is a Node is reused, this can happen when we use Yield(). So if we remove this code , we will get an error eventually as the working directory will not contain the non-rooted paths. 
This was the reason why we use the `WorkingDirectory` property in the first place. We store it when the task starts so we know where we are supposed to be. 
Now what we might be able to do is set the working directory of the `aapt2` tool when we run it. That might work

Reply:
Turns out we already run `aapt2` in the `WorkingDirectory` so we should be ok... in theory.